moabb/paradigms/base.py

```diff
@@ -265,6 +267,13 @@ def get_data(  # noqa: C901
             This pipeline must return an ``np.ndarray``.
             This pipeline must be "fixed" because it will not be trained,
             i.e. no call to ``fit`` will be made.
+        additional_metadata: Literal["default", "all"] | list[str]
```

Comment:
```suggestion
        additional_metadata: None | Literal["all"] | list[str]
```

Reply:
I agree that `None` seems to be more in line with the other kwargs and their defaults. I will change here and in the if statements accordingly